src/custom-element.js

```diff
@@ -899,7 +914,8 @@ function createBaseAmpElementProto(win) {
     assertNotTemplate(this);
     dev().assert(this.isBuilt(),
         'Must be built to receive viewport events');
-    this.dispatchCustomEvent('amp:load:start');
+    this.dispatchCustomEventForTesting('amp:load:start');
+    this.implementation_.isFirstLayoutCompleted_ = this.isFirstLayoutCompleted_;
```

Comment:
i was going to use it in amp-ad layoutCallback,  it is no longer needed. will delete it